### DISCUSSION TITLE
Vercel Account

### DISCUSSION THREAD

**Discussion by Abubakarwantgit** [vercel/community#2094] [Help] [answered]
### Summary

Trying to create an account on Vercel, but got a temporary block error during mobile number verification. The message says to try again after 12 hours. Can anyone help me fix this issue?

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include your username. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about the project you are deploying can help accelerate this process.
